controllers/operator/suite_test.go

```diff
@@ -24,6 +24,7 @@ import (
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	zapLog "go.uber.org/zap"
```

Comment:
Why camelCase? 😄

Reply:
I wanted to use a similar alias to the one used in the telemetry [`suite_test.go`](https://github.com/kyma-project/telemetry-manager/pull/645/files#diff-ac5a219e4af916397378cf78bff3f651a3e0fb56b2deb28aa46869ec0599dba6L27). However, since now this [PR](https://github.com/kyma-project/telemetry-manager/pull/645) is merged, I removed the alias and just used the package name `zap` to be consistent with telemetry [`suite_test.go`](https://github.com/kyma-project/telemetry-manager/pull/645/files#diff-ac5a219e4af916397378cf78bff3f651a3e0fb56b2deb28aa46869ec0599dba6R27)